Students/Stoica S. Razvan Cosmin/Assignment1/asign/src/asign/asign2.java

```diff
@@ -0,0 +1,25 @@
+package asign;
+
+import java.util.Scanner;
+
+public class asign2 {
```

Comment:
According to [this](http://www.oracle.com/technetwork/java/codeconventions-135099.html) class name should have the first letter uppercase